src/main/java/spireMapOverhaul/zones/gremlinTown/powers/TenderizerPower.java

```diff
@@ -0,0 +1,45 @@
+package spireMapOverhaul.zones.gremlinTown.powers;
+
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.WeakPower;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.abstracts.AbstractSMOPower;
+import spireMapOverhaul.zones.gremlinTown.GremlinTown;
+
+import static spireMapOverhaul.util.Wiz.applyToEnemyTop;
+import static spireMapOverhaul.util.Wiz.atb;
+
+public class TenderizerPower extends AbstractSMOPower {
+    public static final String POWER_ID = SpireAnniversary6Mod.makeID("Tenderizer");
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    public TenderizerPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, GremlinTown.ID, PowerType.BUFF,false, owner, amount);
+    }
+
+    @Override
+    public void updateDescription() {
+        description = DESCRIPTIONS[0].replace("{0}", amount + "");
+    }
+
+    public void onAttack(DamageInfo info, int damageAmount, AbstractCreature target) {
+        if (damageAmount > 0 && target instanceof AbstractMonster && info.type == DamageInfo.DamageType.NORMAL
+                && info.owner == owner) {
+            flash();
+            applyToEnemyTop((AbstractMonster) target, new WeakPower(target, amount, false));
```

Comment:
The source here will be the player, is that correct?

Reply:
Yes